easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2.eb

```diff
@@ -11,12 +11,15 @@ source_urls = [GNU_SOURCE]
 
 toolchain = {'name': 'GCC', 'version': '4.9.2'}
 
-dependencies = [
-    ('zlib', '1.2.8'),
-]
+dependencies = [('zlib', '1.2.8')]
+
+builddependencies = [('binutils', '2.25', '', True)]
```

Comment:
Well, GCC can not like it that it has to use a different/older binutils than the one it was compiled with?